Lib/test/test_inspect.py

```diff
@@ -861,8 +861,8 @@ def test_getfullargspec(self):
                                      formatted='(*arg1, arg2=1)')
 
         self.assertFullArgSpecEquals(mod2.annotated, ['arg1'],
-                                     ann_e={'arg1' : list},
-                                     formatted='(arg1: list)')
+                                     ann_e={'arg1' : 'list'},
+                                     formatted="(arg1: 'list')")
```

Comment:
No, I don't like this behavior change. I think we should fix inspect, not the tests (or at least to such an extent).

Reply:
Okay! I deliberately avoided touching `inspect.signature` since I was a little bit confused about what to do there, but fix is incoming for these cases in the next commit